index.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" media="screen" href="css/style.css" />
+    <title>Hello, world!</title>
+</head>
+
+<body>
+    <main id="main-doc">
+        <div class="row">
+            <div class="col-md-2">
+                <nav id="navbar" class="navbar navbar-light bg-light">
```

Comment:
Remember to use semantic tags:
- [ ] Switch  higher level `<div>` tags for `<article>`  tags if they contain any text
- [ ] Switch  lower level `<div>` tags for `<section>`  tags if they contain any text

Reply:
It's Ok here